elasticdl/python/common/model_handler.py

```diff
@@ -111,6 +112,27 @@ def _replace_tf_embedding_column_with_edl(dense_features_layer):
     )
 
 
+def _replace_edl_embedding_column_with_tf(dense_features_layer):
+    new_feature_columns = []
+    for column in dense_features_layer._feature_columns:
+        if isinstance(column, EmbeddingColumn):
+            logger.info(
+                "Replace embedding_column {} from ElasticDL "
+                "version to TF version".format(column.name)
+            )
+            new_column = fc_lib.embedding_column(
+                column.categorical_column, dimension=column.dimension
+            )
+            new_column.set_dense_features_layer_name(dense_features_layer.name)
```

Comment:
EmbeddingColumn from tensorflow don't have this method: set_dense_features_layer_name

Reply:
Done and fix the UT.